npyio_example_test.go

```diff
@@ -89,3 +90,54 @@ func ExampleRead() {
 	// data = ⎡6  1  2⎤
 	//        ⎣3  4  5⎦
 }
+
+func ExamplePartialRead() {
+	out, err := os.Create("data.npy")
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	f := []float64{0, 1, 2, 3, 4, 5}
+	fmt.Printf("-- original data --\n")
+	fmt.Printf("data = %v\n", f)
+	err = npyio.Write(out, f)
+	if err != nil {
+		log.Fatalf("error writing data: %v\n", err)
+	}
+	out.Close()
```

Comment:
as we are writing data, we should check the error (if any) of the call to `out.Close()`.